{
  "modified": "2024-10-24T01:01:58Z",
  "published": "2024-06-25T13:36:26Z",
  "schema_version": "1.5.0",
  "id": "MAL-2024-5266",
  "summary": "Malicious code in insanepackage217424422342983 (PyPI)",
  "details": "\n---\n_-= Per source details. Do not edit below this line.=-_\n",
  "affected": [
    {
      "package": {
        "ecosystem": "PyPI",
        "name": "insanepackage217424422342983",
        "purl": "pkg:pypi/insanepackage217424422342983"
      },
      "versions": [
        "1.0.0"
      ]
    }
  ],
  "credits": [
    {
      "name": "ReversingLabs",
      "type": "FINDER",
      "contact": [
        "https://www.reversinglabs.com"
      ]
    }
  ],
  "database_specific": {
    "malicious-packages-origins": [
      {
        "source": "reversing-labs",
        "sha256": "aeb87b4c0db73719d4b31e083b2d606e72c37b47635f8e7dda53bee82271f6ba",
        "import_time": "2024-06-28T02:49:14.555002964Z",
        "id": "RLMA-2024-04048",
        "modified_time": "2024-06-25T13:36:26Z",
        "versions": [
          "1.0.0"
        ]
      },
      {
        "source": "reversing-labs",
        "sha256": "9da7f6b4550228a5204992f84a92b5047fd072bb3ebe379fa438549285abe175",
        "import_time": "2024-10-24T00:59:20.762080506Z",
        "id": "RLUA-2024-08408",
        "modified_time": "2024-10-16T14:42:34Z"
      }
    ]
  }
}
